src/utils/hash/classHash.ts

```diff
@@ -110,7 +133,9 @@ export function formatSpaces(json: string) {
 
 /**
  * Compute hinted class hash for legacy compiled contract (Cairo 0)
- * @returns format: hex-string
+ * @param {LegacyCompiledContract} compiledContract
+ * @returns {string} hex-string
+ * No example, as this function is not exported to user
  */
 export default function computeHintedClassHash(compiledContract: LegacyCompiledContract) {
```

Comment:
Can we remove default from here? Would be more clean with explicit imports, this func isn't immediately obvious to be the default one, especially since we have other ones like computeLegacyContractClassHash

Reply:
Traced in #1135